incubator/oauth-proxy/README.md

```diff
@@ -73,4 +75,21 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 $ helm install incubator/oauth-proxy --name my-release -f values.yaml
 ```
 
+### Configuration of htpasswd config file
+
+If you want to, in adition to oauth2, have some basic auth working, specify the `htpasswd` section.
```

Comment:
Whats the preference / sequence of oath vis a vis basic auth? In my view, it would be good to add a small note here about that.

Reply:
I think it has a look at the oauth2 cookie first and if it doesn't find it then it goes and looks for basic auth, as per the code from: https://github.com/bitly/oauth2_proxy/blob/d75f626cdd664fa75552717c4abd3018877d3b2c/oauthproxy.go#L602

I will add a note here.